imcsdk/apis/server/bios.py

```diff
@@ -106,8 +117,13 @@ def _get_device(parent_dn, device_type, device_name):
     elif _is_boot_order_policy(parent_dn):
         if device_type not in policy_device_dict:
             return None
-        class_struct = load_class(policy_device_dict[device_type])
-        class_obj = class_struct(parent_mo_or_dn=parent_dn)
+        class_struct = load_class(policy_device_dict[device_type]["class_id"])
+        access = policy_device_dict[device_type]["access"]
+        if device_type in ["cdrom", "fdd"]:
+            class_obj = class_struct(parent_mo_or_dn=parent_dn, access=access)
```

Comment:
@waghswapnil can you add a comment here for reference in the future. need to explain, why cdrom and fdd needed special handling.